rest_framework_extensions/cache/mixins.py

```diff
@@ -8,20 +8,22 @@ class BaseCacheResponseMixin(object):
     # test_cache_reponse(view_instance, method, should_rebuild_after_method_evaluation)
     object_cache_key_func = extensions_api_settings.DEFAULT_OBJECT_CACHE_KEY_FUNC
     list_cache_key_func = extensions_api_settings.DEFAULT_LIST_CACHE_KEY_FUNC
+    object_cache_timeout = extensions_api_settings.DEFAULT_CACHE_RESPONSE_TIMEOUT
+    list_cache_timeout = extensions_api_settings.DEFAULT_CACHE_RESPONSE_TIMEOUT
 
 
 class ListCacheResponseMixin(BaseCacheResponseMixin):
-    @cache_response(key_func='list_cache_key_func')
+    @cache_response(key_func='list_cache_key_func', timeout='list_cache_timeout')
```

Comment:
i am getting a `ValueError: invalid literal for int() with base 10: 'list_cache_timeout'`.
To fix it i used `@cache_response(key_func='list_cache_key_func', timeout=list_cache_timeout)` instead.